data-manipulation/encryption/create-new-key-via-cryptacquirecontext.yml

```diff
@@ -0,0 +1,26 @@
+rule:
+  meta:
+    name: create new key via CryptAcquireContext
+    namespace: data-manipulation/encryption
+    author: chuong.dong@fireeye.com
+    scope: function
+    att&ck:
+      - Defense Evasion::Obfuscated Files or Information [T1027]
+    mbc:
+      - Cryptography::Encryption Key [C0028]
+    references:
+      - https://docs.microsoft.com/en-us/windows/win32/api/wincrypt/nf-wincrypt-cryptacquirecontexta
+    examples:
+      - E5369AC309F1BE6D77AFEEB3EDAB0ED8:0x401350
+      - 849945BB1D222607446C7A917F0D935A:0x405482
```

Comment:
since we have one good example in testfiles, let's remove this.

our CI system runs the linter against all examples listed in this section, so by adding too many extra examples, performance gets worse.

if there's something notable about this particular sample, definitely keep it. otherwise, its a minor performance tweak.